src/App.js

```diff
@@ -30,14 +30,17 @@ class App extends Component {
   handleSubmit(event) {
     if(!quotes.includes(this.state.value)){
       quotes.push(this.state.value);
-      this.setState({ quote: this.getQuote()});
+      this.renderNewQuote();
 
       console.log(this.state.quote )
     }
     this.persistQuotes();
     event.preventDefault();
   }
   
+  renderNewQuote(event) {
+    this.setState({ quote: this.getQuote()});
+  }
```

Comment:
you sometimes leave a blank line in between your methods, and sometimes don't.  It's good to pick one and be consistent about it.  I favor the blank line.